main.go

```diff
@@ -82,7 +82,7 @@ func createConfigsModelFromEnvs() ConfigsModel {
 		ForceProvisioningProfileSpecifier: os.Getenv("force_provisioning_profile_specifier"),
 		ForceProvisioningProfile:          os.Getenv("force_provisioning_profile"),
 		ForceCodeSignIdentity:             os.Getenv("force_code_sign_identity"),
-		CustomExportOptionsPlistContent:   os.Getenv("custom_export_options_plist_content"),
+		CustomExportOptionsPlistContent:   strings.TrimSpace(os.Getenv("custom_export_options_plist_content")),
```

Comment:
do not do this here! This way we will not print the current value of the user input. 
Do the trimm [here](https://github.com/bitrise-io/steps-xcode-archive/pull/84/files#diff-7ddfb3e035b42cd70649cc33393fe32cR348), by wiring out a new var: `customExportOptionsPlistContent`, which holds the trimmed value and use this var.